src/main/java/org/folio/edge/controller/AuthenticationController.java

```diff
@@ -26,8 +22,7 @@ public class AuthenticationController implements AuthenticationApi {
 
   @Override
   @PostMapping("/token")
-  public ResponseEntity<AccessTokenResponse> getToken(@NotNull @Pattern(regexp = "client_credentials") @Valid String grantType,
-                                                      @NotNull @Pattern(regexp = "innreach_tp") @Valid String scope,
+  public ResponseEntity<AccessTokenResponse> getToken(String grantType, String scope,
                                                       String xOkapiTenant, String xOkapiToken, String authorization) {
```

Comment:
Before this place was giving error like this
```
17:25:40 [] [] [] [] INFO  SecurityService      Set okapi token in the header.
17:25:40 [] [test_tenant] [] [edge-inn-reach] ERROR ionHandlerController Unexpected exception: HV000151: A method overriding another method must not redefine the parameter constraint configuration, but method AuthenticationController#getToken(String, String, String, String, String) redefines the configuration of AuthenticationApi#getToken(String, String, String, String, String).
jakarta.validation.ConstraintDeclarationException: HV000151: A method overriding another method must not redefine the parameter constraint configuration, but method AuthenticationController#getToken(String, String, String, String, String) redefines the configuration of AuthenticationApi#getToken(String, String, String, String, String).
	at org.hibernate.validator.internal.metadata.aggregated.rule.OverridingMethodMustNotAlterParameterConstraints.apply(OverridingMethodMustNotAlterParameterConstraints.java:24) ~[hibernate-validator-8.0.0.Final.jar:8.0.0.Final]
	at org.hibernate.validator.internal.metadata.aggregated.ExecutableMetaData$Builder.assertCorrectnessOfConfiguration(ExecutableMetaData.java:462) ~[hibernate-validator-8.0.0.Final.jar:8.0.0.Final]
	at org.hibernate.validator.internal.metadata.aggregated.ExecutableMetaData$Builder.build(ExecutableMetaData.java:380) ~[hibernate-validator-8.0.0.Final.jar:8.0.0.Final]
	at org.hibernate.validator.internal.metadata.aggregated.BeanMetaDataBuilder$BuilderDelegate.build(BeanMetaDataBuilder.java:260) ~[hibernate-validator-8.0.0.Final.jar:8.0.0.Final]
	at org.hibernate.validator.internal.metadata.aggregated.BeanMetaDataBuilder.build(BeanMetaDataBuilder.java:133) ~[hibernate-validator-8.0.0.Final.jar:8.0.0.Final]
	at org.hibernate.validator.internal.metadata.BeanMetaDataManagerImpl.createBeanMetaData(BeanMetaDataManagerImpl.java:206) ~[hibernate-validator-8.0.0.Final.jar:8.0.0.Final]
	at org.hibernate.validator.internal.metadata.BeanMetaDataManagerImpl.getBeanMetaData(BeanMetaDataManagerImpl.java:165) ~[hibernate-validator-8.0.0.Final.jar:8.0.0.Final]
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateParameters(ValidatorImpl.java:267) ~[hibernate-validator-8.0.0.Final.jar:8.0.0.Final]
	at org.hibernate.validator.internal.engine.ValidatorImpl.validateParameters(ValidatorImpl.java:235) ~[hibernate-validator-8.0.0.Final.jar:8.0.0.Final]
	at org.springframework.validation.beanvalidation.MethodValidationInterceptor.invoke(MethodValidationInterceptor.java:121) ~[spring-context-6.0.4.jar:6.0.4]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.0.4.jar:6.0.4]
```

I decide to remove constraints because of it already available in AuthenticationApi class 
![image](https://user-images.githubusercontent.com/113523904/214563256-fda7262d-aae8-4011-bb04-2ec3f4b73f7b.png)

Reply:
This issue is a result of different annotations in params you can try remove @Valid annotation

Reply:
It is working only there is not any params. If I remove only @Valid annotation, tests are failing. So I removed all params

Reply:
ok It is resolved